### PR TITLE
adding TXGammaTRandoms stage

### PR DESCRIPTION
**Pull request by ih64**
I mostly copy pasted TXGammaDimStars and then changed it around so it loads the random catalog instead of the star catalog. Not sure if I handled the sacc outputs correctly, but happy to go back and make changes

### CODE REVIEW

**Comment by ih64** [comment]
forgot to mention this is aimed at issue #71 , one of the items in the issue is to do gamma_t/x around randoms

File: txpipe/twopoint.py [hunk 1]
```diff
@@ -1429,6 +1429,164 @@ def write_output_sacc(self, data, meta, results):
...
         # Also make a plot of the data
 
+class TXGammaTRandoms(TXTwoPoint):
+    """
+    This subclass of the standard TXTwoPoint uses the centers
+    of stars as "lenses", as a systematics test.
+    """
+    name = "TXGammaTRandoms"
+    inputs = [
+        ('shear_catalog', MetacalCatalog),
```
Comment (joezuntz) [inline txpipe/twopoint.py:1439]: I think some of these aren't needed for this?

File: txpipe/twopoint.py [hunk 2]
```diff
@@ -1429,6 +1429,164 @@ def write_output_sacc(self, data, meta, results):
...
+        'calcs':[0,1,2],
+        'min_sep':2.5,
+        'max_sep':100,
+        'nbins':20,
+        'bin_slop':0.1,
+        'sep_units':'arcmin',
+        'flip_g2':True,
+        'cores_per_task':20,
+        'verbose':1,
+        'reduce_randoms_size':1.0,
```
Comment (joezuntz) [inline txpipe/twopoint.py:1463]: Are we still subtracting randoms?  Not sure if that makes sense in this context!  This will be in the superclass.

**Comment by joezuntz** [comment]
Just asked a couple of questions - thanks1

**Pull request merged**